### ISSUE TITLE
L2 regularization 

### ISSUE THREAD

**Issue by ttgump** [yihong-chen/neural-collaborative-filtering#14]
Hi,
I notice that in your config code of mlp and neumf, you set l2_regularization. But I can't find l2 regularization in your model and loss implementation. Could you help me to know how to implement the l2 regularization? Thanks.

**Comment by yihong-chen** [comment]
Hi, the `l2_regularization` is simply the weight decay hyper-parameter for the optimizer. I hope this helps.
